### DISCUSSION TITLE
nvim-spectre errors out during replaceall

### DISCUSSION THREAD

**Discussion by kapral18** [LazyVim/LazyVim#1272] [Q&A]
when I invoked spectre replaceall with \<leader\>R I see this

![CleanShot 2023-08-01 at 23 48 50@2x](https://github.com/LazyVim/LazyVim/assets/1625373/aa5b7162-5745-4735-bd46-9d951203ad46)

here is my lazyvim config https://github.com/kapral18/config.nvim

**Comment by kapral18** [comment]
I think I found the issue :D It only happens when the explorer is open, but it still sucks

**Comment by kapral18** [comment]
up

**Reply by dpetka2001** [reply]
You might want to raise an issue at `spectre`'s repo to see what can be done. I'm sure they will more useful. It would be good if they could provide an option to exclude certain buffer filetypes, but I don't see such an option from little that i looked around. Maybe they will have a better idea how to achieve this.
